tests/torch/test_nncf_network.py

```diff
@@ -903,8 +903,9 @@ def get_ip_graph_for_test(nncf_graph: NNCFGraph,
             ip = PreHookInsertionPoint(node.node_name, in_edge.input_port_id)
             pre_hooks.append(ip)
 
-        ip = PostHookInsertionPoint(node.node_name)
-        post_hooks.append(ip)
+        if node.node_type != 'split':
```

Comment:
Better to handle this consistently with the regular NNCFNetwork (nncf/torch/nncf_network.py:551, `get_insertion_point_graph` function)